test/e2e/azure/cli.go

```diff
@@ -290,7 +289,9 @@ func (a *Account) CreateDeployment(name string, e *engine.Engine) error {
 	if err != nil {
 		log.Printf("\nError from deployment for %s in resource group %s:%s\n", d.Name, a.ResourceGroup.Name, err)
 		log.Printf("Command Output: %s\n", out)
-		if bytes.Contains(out, []byte("PrincipalNotFound")) {
+		// Keeping this retry scenario in here for historical purposes.
```

Comment:
Can't we just remove this code instead? It's always in git if we need to revert...